### DISCUSSION TITLE
Cy.visit will not work when the url is having # and exits from test please provide the workaround

### DISCUSSION THREAD

**Discussion by shobhag980** [cypress-io/cypress#24989] [Questions and Help]
cy.visit is not working when your url is having '#' it breaks the test while loading and exits you from test. saying"you seem to have gotten lost try one of the links instead" again if you navigate to your specrunner and execute your test same thing happens when i tried same with selenium i was able to load the page. cypress is not able to open the page

i tried location.hash, cypress wait until function and many other solutions provided through communities but nothing worked in my case

**Comment by shobhag980** [comment]
and also if i try with cypres version less then 10 i am getting error Error: EISDIR: illegal operation on a directory, 
also tried the resolution provided Error: EISDIR: illegal operation on a directory #1244 but still issue not fixed

**Reply by jjhelguero** [reply]
You can visit a url with a hash. I was able to visit `https://docs.cypress.io/api/commands/contains#Yields` with cypress v10.4.0.

Do you have a reproducible example for your scenario?
